tests/app/submitter/conftest.py

```diff
@@ -83,12 +160,37 @@ def fake_questionnaire_store(fake_metadata, fake_response_metadata):
 
     store.answer_store = AnswerStore()
     store.answer_store.add_or_update(user_answer)
-    store.metadata = fake_metadata
+    store.metadata = MetadataProxy.from_dict(fake_metadata)
     store.response_metadata = fake_response_metadata
 
     return store
 
 
+@pytest.fixture
+def fake_questionnaire_store_no_ref_p_end_date():
```

Comment:
I had to make a new one as otherwise would need to delete/remove `ref_p_end_date` from the metadata proxy object for the `test_ref_period_end_date_is_not_in_output` in test_converter